pkg/ovs/openflow/default.go

```diff
@@ -0,0 +1,24 @@
+// +build !windows
```

Comment:
Add a line between the "build" tag and license code.

Reply:
Done